lib/collectionspace/converter/default/record.rb

```diff
@@ -1,6 +1,7 @@
 module CollectionSpace
   module Converter
     module Default
+      ::Default = CollectionSpace::Converter::Default
```

Comment:
I think we can remove this line. If there's a weird load order issue we can use require_relative to include default.rb first.

Reply:
Ok, wasn't sure what that did, but just copied it over from your alternate PR. 😆